### PR TITLE
[T932][FIX] Moved Attedence position under summery tab

### PR DESCRIPTION
**Pull request by YogeshMahera-SerpentCS**
TASK #[932](https://www.quartile.co/web#view_type=form&model=project.task&id=932&active_id=932&menu_id=)

### CODE REVIEW

File: hr_attendance_timesheet_adj/views/hr_timesheet_sheet_sheet_views.xml [hunk 1]
```diff
@@ -21,6 +21,24 @@
                     </div>
                 </button>
             </xpath>
+            <xpath expr="//page[3]" position="replace">
+            </xpath>
```
Comment (yostashiro) [inline hr_attendance_timesheet_adj/views/hr_timesheet_sheet_sheet_views.xml:25]: Can you please check if making the page invisible would also work (i.e. present all the attendance records correctly in the first page). If that is the case, we would like to go for that option in preference of not using replace.  If invisible attribute does not work then we will go with replace.

**Review by yostashiro — COMMENTED** [review]
@YogeshMahera-SerpentCS Please bump the module version. Please follow up on my comment, merge this PR, and update the staging server.

File: hr_attendance_timesheet_adj/views/hr_timesheet_sheet_sheet_views.xml (hunk 1 above)
Comment (YogeshMahera-SerpentCS) [reply hr_attendance_timesheet_adj/views/hr_timesheet_sheet_sheet_views.xml:25]: Hi @yostashiro It'seems not working with invisible.
Peek 2020-04-13 12-18

**Comment by TimLai125** [comment]
@YogeshMahera-SerpentCS Please rebase your feature branch and fix the TravisCI build

**Comment by YogeshMahera-SerpentCS** [comment]
Hi, @TimLai125 I have rebased but seems still fail.

**Pull request closed without merging**